internal/app/tfsec/rules/aws/vpc/disallow_mixed_sgr_rule.go

```diff
@@ -0,0 +1,74 @@
+package vpc
+
+import (
+	"fmt"
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/block"
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/debug"
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/scanner"
+	"github.com/aquasecurity/tfsec/pkg/provider"
+	"github.com/aquasecurity/tfsec/pkg/result"
+	"github.com/aquasecurity/tfsec/pkg/rule"
+	"github.com/aquasecurity/tfsec/pkg/severity"
+)
+
+func init() {
+	scanner.RegisterCheckRule(rule.Rule{
+		Provider:  provider.AWSProvider,
+		Service:   "vpc",
+		ShortCode: "disallow-mixed-sgr",
+		Documentation: rule.RuleDocumentation{
+			Summary:     "Ensures that usage of security groups with inline rules and security group rule resources are not mixed.",
+			Explanation: `
+Mixing Terraform standalone security_group_rule resource and security_group resource with inline ingress/egress rules results in rules being overwritten during Terraform apply.
+`,
+			Impact:      "Security group rules will be overwritten and will result in unintended blocking of network traffic",
+			Resolution:  "Either define all of a security group's rules inline, or none of the security group's rules inline",
+			BadExample: []string{`
+resource "aws_security_group_rule" "bad_example" {
+  	security_group_id = aws_security_group.bad_example_sg.id
+	type = "ingress"
+	cidr_blocks = ["172.31.0.0/16"]
+}
+
+resource "aws_security_group" "bad_example_sg" {
+	ingress {
+		cidr_blocks = ["10.0.0.0/16"]
+	}
+}
+`},
+			GoodExample: []string{`
+resource "aws_security_group_rule" "good_example" {
+  	security_group_id = aws_security_group.good_example_sg.id
+	type = "ingress"
+	cidr_blocks = ["10.0.0.0/16", "172.31.0.0/16"]
+}
+
+resource "aws_security_group" "good_example_sg" {
+}
+`},
+			Links: []string{
+				"https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group",
```

Comment:
Can we change this link and target the message specifically 
```suggestion
				"https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group#resource-aws_security_group",
```

Reply:
Ack. Linked changed. Thank you.